src/main/java/core/basesyntax/strategy/ReturnOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperation implements OperationHandler {
+    private final Storage storage;
+
+    public ReturnOperation(Storage storage) {
+        this.storage = storage;
+    }
+
+    @Override
+    public void count(FruitTransaction transaction) {
+        if (transaction.getQuantity() < 0) {
+            throw new RuntimeException("Quantity can't be less than 0");
+        }
+        storage.addFruit(transaction.getFruit(), transaction.getQuantity());
+    }

```

Comment:
This class should not have a direct dependency on `Storage`. Instead, consider using dependency injection to pass a specific instance where needed, which will make your class more flexible and testable.